fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -941,13 +937,6 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_VECTORIZED_ENGINE, varType = VariableAnnotation.REMOVED)
     public boolean enableVectorizedEngine = true;
 
-    @VariableMgr.VarAttr(name = ENABLE_PIPELINE_ENGINE, fuzzy = true, needForward = true,
-            varType = VariableAnnotation.EXPERIMENTAL)
-    private boolean enablePipelineEngine = true;
-
-    @VariableMgr.VarAttr(name = ENABLE_PIPELINE_X_ENGINE, fuzzy = false, varType = VariableAnnotation.EXPERIMENTAL)
-    private boolean enablePipelineXEngine = true;
-
```

Comment:
do not remove them, use varAttr `varType = VariableAnnotation.REMOVED` for backward compatibility